tt: add page (#10760)
tldr-pages/tldr: pages/linux/tt.md

# tt

> > A terminal based typing tester.

Reviewer: ```suggestion
> A terminal based typing test.
```

Author: Updated

> More information: <https://github.com/lemnos/tt>.

> - Start quote mode with builting quote list in english 'en':

Reviewer: ```suggestion
- Start quote mode with the builtin quote list in English:
```

Author: Updated

> `tt -quotes en`

Reviewer: ```suggestion
`tt -quotes {{en}}`
```

Author: Updated

- Produce a test consisting of 50 randomly drawn words in 5 groups of 10 words each:

> `tt -n 10 -g 5`

Reviewer 2: ```suggestion
`tt -n {{10}} -g {{5}}`
```

Author: Updated

- Start a timed test lasting 10 seconds:

> `tt -t 10`

Reviewer 2: ```suggestion
`tt -t {{10}}`
```

Author: Updated

> - Start tt with no theming and showing your WPM as you type:

Reviewer: ```suggestion
- Start `tt` with no theming and showing your WPM as you type:
```

Author: Updated
